test/test_xnnpack_integration.py

```diff
@@ -0,0 +1,347 @@
+from __future__ import division
+
+import torch
+from torch.nn import functional as F
+import torch.testing._internal.hypothesis_utils as hu
+from torch.testing._internal.common_utils import TestCase, run_tests
+from torch.testing import FileCheck
+from hypothesis import given, assume
+from hypothesis import strategies as st
+import unittest
+import io
+
+class TestXNNPACKOps(TestCase):
+    @given(data_shape=hu.array_shapes(2, 4, 2, 64),
+           weight_output_dim=st.integers(2, 64),
+           use_bias=st.booleans())
+    def test_linear(self, data_shape, weight_output_dim, use_bias):
+        input_data = torch.rand(data_shape)
+        weight = torch.rand((weight_output_dim, data_shape[-1]))
+        if use_bias:
+            bias = torch.rand((weight_output_dim))
+        else:
+            bias = None
+        ref_result = F.linear(input_data, weight, bias)
+        packed_weight_bias = torch.ops.xnnpack.linear_prepack(weight, bias)
+        output_linear_xnnpack = torch.ops.xnnpack.linear_packed(input_data, packed_weight_bias)
+        torch.testing.assert_allclose(ref_result, output_linear_xnnpack, rtol=1e-2, atol=1e-3)
+
+
+    @given(batch_size=st.integers(1, 3),
```

Comment:
Please be sure to test Linear and Conv with batch size = 0.  (It doesn't have to be part of this same general test.  Just test that you can run it and get a proper shape output.)